src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.lang.reflect.Array;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Application {
+    FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
The 'figureSupplier' instance variable should be private to encapsulate the internal representation of the class.